pages/docs/tutorials/kafka/managing-schemas-using-schema-registry.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Managing schemas using Schema Registry
+description: The guide explains how you can manage schemas using Schema Registry.
+weight: 50
+---
+
+## Introduction
+You will learn how to manage schemas with Schema Registry. Managing schemas will enable us to ensure a consistent data exchange across distributed systems, thus minimizing errors.
+
+## Background context
+The need for schema management has become increasingly prevalent to handle the evolving complexity of modern Event-Driven Architecture. A Schema Registry is a centralized service that stores and maintains the schemas for data exchanged between various components of a modern distributed system. Validating data being exchanged helps in maintaining data consistency and compatibility. 
+
+While several Schema Registry implementations exist, you will use the Apicurio Registry in this tutorial. Apicurio Registry is a popular open-source Schema Registry implementation that supports multiple serialization formats and facilitates schema management for diverse data in distributed systems. You will be using the Apicurio Registry in combination with Avro, which is a language-neutral data serialization system.
+
+## Installation guide
+1. Install [Docker](https://docs.docker.com/engine/install/) from the official website.
+
+## Steps
+1. Run Apicurio Registry Docker Image in a Docker container by using the command:
+    <CodeBlock language="bash">
+    {`docker run -it -p 8080:8080 apicurio/apicurio-registry-mem:2.5.7.Final`}
+    </CodeBlock>
+2. Once your Apicurio Registry is running, you can upload the schema in another terminal window. Create an Avro Schema Artifact using the following command:
+<CodeBlock>
+curl \
+http://localhost:8080/apis/registry/v2/groups/my-group/artifacts \
+-X POST  \
+-H "Content-Type: application/json; artifactType=AVRO" \
+-H "X-Registry-ArtifactId: UserSignedUp" \
+--data @- << EOF
+{
+  "type": "record",
+  "name": "UserSignedUp",
+  "namespace": "com.company",
+  "doc": "User sign-up information",
+  "fields": [
+    {
+      "name": "userId",
+      "type": "int"
+    },
+    {
+      "name": "userEmail",
+      "type": "string"
+    }
+  ]
+}
+EOF
+</CodeBlock>
+3. You can download the Avro schema by visiting the URL below and replacing {UserSignedUp} with the id of the schema used in the previous step.
```

Comment:
You can download the Avro schema by visiting the URL below:
 http://localhost:8080/apis/registry/v2/groups/my-group/artifacts/UserSignedUp
 
 @derberg Is this what you were looking for? 
 Also, since I'm adding it as a note, should this not be a separate step in the tutorial?

Reply:
yes, just a note under the step that is responsible for uploading schema

it should not be a step as it is just an info - nobody has to execute this step

steps are things that user must follow to go from 1 to 10

Reply:
alternative - make it a step but somehow mark as optional

you can consult with other tech writers

Reply:
My vote is to change this tutorial step to instead be a `Remember` block. (a note, as Lukasz also initially suggested)

@Arya-Gupta, please go ahead and change it into a `Remember` block and make sure to include the full URL, as Lukasz also noted. You should end up with something like the following:

```
<Remember>
Download your Avro schema by visiting the following URL: 
http://localhost:8080/apis/registry/v2/groups/my-group/artifacts/UserSignedUp.
</Remember>
```

Reply:
I've changed it to a `Remember` block!